easybuild/easyconfigs/m/MEME/MEME-5.4.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,39 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'MEME'
+version = '5.4.1'
+
+homepage = 'https://meme-suite.org/meme/index.html'
+description = """The MEME Suite allows you to: * discover motifs using MEME, DREME (DNA only) or
+ GLAM2 on groups of related DNA or protein sequences, * search sequence databases with motifs using
+ MAST, FIMO, MCAST or GLAM2SCAN, * compare a motif to all motifs in a database of motifs, * associate
+ motifs with Gene Ontology terms via their putative target genes, and * analyse motif enrichment
+ using SpaMo or CentriMo."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
```

Comment:
`MEME` claims to have `MPI` support, is there a particular reason for building without it?

Reply:
Hmm, no particular reason, I could look into building `meme` with MPI support instead, which probably boils down to using `gompi` as toolchain than `GCC`...

Reply:
fixed in 752d645